view/theme/frio/templates/wall_thread.tpl

```diff
@@ -125,16 +125,11 @@ as the value of $top_child_total (this is done at the end of this file)
 			{{* contact info header*}}
 		<div class="contact-info">
 			<div class="preferences">
-				{{if $item.network_icon != ""}}
-					<span class="wall-item-network"><i class="fa fa-{{$item.network_icon}}" title="{{$item.network_name}}" aria-hidden="true"></i></span>
+				{{if $item.network_icon != "" && $item.plink}}
+					<span class="wall-item-network"><a href="{{$item.plink.href}}" class="plink u-url" aria-label="{{$item.plink.title}}" target="_blank"><i class="fa fa-{{$item.network_icon}} fakelink" title="{{$item.network_name}} - {{$item.plink.title}}" aria-hidden="true"></i></span>
```

Comment:
The conditions on `$item.network_icon` and `$item.plink` were separate, please keep them separate in the new behavior:
- network icon and plink: The case you created
- network icon but no plink: Just the network icon like before
- plink but no network icon: Just the external link like before
- no network icon nor plink: Nothing, like before.

Reply:
I handled the new case with a new if query and wrote the remaining code in an else

Reply:
Thanks, I now realized I missed the default network display when there's no icon. It should still be a link to the original post if a plink exists.

Reply:
that should be already there?
The code on this is exactly the same ase before, i just wrapped it in an if for the new case.
Or am i getting it wrong? :)

Reply:
Let's go back to our four cases:
1. Network Icon and PLink: A link to the original post showing the network icon.
2. No Network Icon but PLink: A link to the original post showing the network name.
3. Network Icon but no PLink: Just the network icon.
4. Neither Network Icon not PLink: Just the network name.

At the moment I believe case 2 isn't handled expectedly.